app/assets/stylesheets/welcome.scss

```diff
@@ -1,12 +1,10 @@
-$primary-color: black;
-$secondary-color: pink;
-$dark-gray: #333839;
-$light-gray: #D7D8D8;
+@import './colors';
+//= depend_on_asset "welcome-background.jpg"
```

Comment:
be careful about this comment, especially with `//=` because it can be interpreted by sprockets as a dependency

Reply:
It is supposed to be a dependency. I had begun to set up the asset pipeline to work on Heroku. Is that okay?